README.md

```diff
@@ -39,6 +39,7 @@ The following matrix indicates the version of Trivy and Trivy adapter installed
 
 | Harbor          | Trivy Adapter | Trivy           |
 |-----------------|---------------|-----------------|
+| [harbor v2.4.1] | v0.25.0       | [trivy v0.21.3] |
```

Comment:
Usually we update the compatibility matrix once the adapter service is released and version is bumped in the https://github.com/goharbor/harbor repo.